### DISCUSSION TITLE
How'd this person join GitHub in 1971?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52463] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

[The profile I'm talking about](https://github.com/G-eb)

**Comment by airtower-luna** [comment, marked as answer]
They definitely didn't, because neither Git nor GitHub existed back then. :wink: 

Most likely they simply created some commits with dates in 1971 as the author date. `git commit` has a [`--date` option](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---dateltdategt) to override the date, so it's not hard.
